src/server.js

```diff
@@ -0,0 +1,152 @@
+const _ = require('lodash/fp')
+const express = require('express')
+const cors = require('cors')
+const uuid = require('uuid/v4')
+const { google } = require('googleapis')
+const { Firestore } = require('@google-cloud/firestore')
+const bodyParser = require('body-parser')
+const Joi = require('@hapi/joi')
+const { promiseHandler, Response, validateInput } = require('./utils')
+
+
+const main = async () => {
+  const auth = new google.auth.GoogleAuth({
+    scopes: ['https://www.googleapis.com/auth/cloud-platform']
+  })
+  const authClient = await auth.getClient()
+  const iam = google.iam({ version: 'v1', auth: authClient })
+  const iamcredentials = google.iamcredentials({ version: 'v1', auth: authClient })
+
+  const firestore = new Firestore()
+
+  const app = express()
+  app.use(bodyParser.json())
+  app.use(cors())
+  app.use('/docs', express.static('docs'))
+
+  const emailSchema = Joi.string().email()
+
+  const projectId = 'terra-test-users-1'
+
+  const deleteUser = async email => {
+    try {
+      await iam.projects.serviceAccounts.delete({
+        name: `projects/${projectId}/serviceAccounts/${email}`
+      })
+    } catch (error) {
+      // If it doesn't exist, silently succeed
+      if (error.code !== 404) {
+        throw error
+      }
+    }
+    await firestore.doc(`users/${email}`).delete()
+  }
+
+  const withAuth = wrappedFn => async (req, ...args) => {
+    const idToken = (req.headers.authorization || '').split(' ')[1]
+    const ticket = await authClient.verifyIdToken({ idToken, audience: 'https://terra-lyle.appspot.com' })
+    const { email } = ticket.getPayload()
+    if (email !== 'lyle-user@terra-lyle.iam.gserviceaccount.com') {
+      throw new Response(403)
+    }
+    return wrappedFn(req, ...args)
+  }
+
+  /**
+   * @api {get} /status System status
+   * @apiName status
+   * @apiVersion 1.0.0
+   * @apiGroup System
+   * @apiSuccess (Success 200) -
```

Comment:
Annoying that we have to specify something, but it appears that's just how it is.